src/ValidationPackages/ValidationPackagesConfig.fs

```diff
@@ -0,0 +1,42 @@
+﻿namespace ARCtrl.ValidationPackages
+
+open ARCtrl.Helper
+open Fable.Core
+
+[<AttachMembers>]
+type ValidationPackagesConfig(validation_packages, ?arc_specification) =
+    
+    let mutable _validation_packages : ValidationPackage array = validation_packages
+    let mutable _arc_specification : string option = arc_specification
+
+    member this.ValidationPackages 
+        with get() = _validation_packages
+        and set(validation_packages) = _validation_packages <- validation_packages
+
+    member this.ARCSpecification
+        with get() = _arc_specification
+        and set(arc_specification) = _arc_specification <- arc_specification
+
+    static member make validation_packages arc_specification = ValidationPackagesConfig(validation_packages=validation_packages, ?arc_specification=arc_specification)
+
+    static member create(validation_packages,?arc_specification) : ValidationPackagesConfig =
+        ValidationPackagesConfig.make validation_packages arc_specification
+    
+    static member toString (vp : ValidationPackagesConfig) =
```

Comment:
This does not return string?

Reply:
I was mirroring the API of Comment here: 
https://github.com/nfdi4plants/ARCtrl/blob/403081ee9648a87ffd0198baaebef025df51b006/src/Core/Comment.fs#L27-L28
But i do not need this function at all so i'll just remove it until it is needed